tests/src/python/test_qgspallabeling_server.py

```diff
@@ -142,10 +142,13 @@ def sync_map_settings(self):
         """
         Sync custom test QgsMapSettings to Project file
         """
+        pass
```

Comment:
What's happening here? I think this should be removed or `@unittest.skip('this test is temporarily disabled because X')` be used instead.